src/js/options.js

```diff
@@ -1,6 +1,6 @@
-import extension from './extension.js';
+import extension from '/js/extension.js';
```

Comment:
In firefox I get an error on this line when opening the `welcome.html` page:

```
Uncaught SyntaxError: import declarations may only appear at top level of a module
```

Reply:
Fixed